scripts/apps/search/MultiImageEdit.ts

```diff
@@ -204,6 +206,32 @@ export function MultiImageEditDirective(asset, $sce) {
                     scope.selectImage(image);
                 }
             };
+
+            scope.copyMetadata = (metadata) => {
+                scope.metadataFromStorage = metadata;
+                localStorage.setItem('metadata:items', JSON.stringify(metadata));
+            };
+
+            scope.toggleAutofill = () => {
+                scope.autoFill = !scope.autoFill;
+                localStorage.setItem('metadata:autofill', scope.autoFill);
+            };
+
+            scope.pasteMetadata = () => {
+                scope.metadata = JSON.parse(localStorage.getItem('metadata:items'));
+                _.forEach(scope.metadata, (metadata, key) => {
+                    scope.onChange(key);
+                });
+            };
+
+            scope.clearSavedMetadata = () => {
+                localStorage.removeItem('metadata:items');
+                delete scope.metadataFromStorage;
```

Comment:
seems like you only use this to indicate if it's empty, imo would be better to populate it like:
```
scope.metadataFromStorage = !!localStorage.getItem('metadata:items');
```
and here you can just set it to `false`, so it doesn't compare the whole metadata json just to check it's empty

Reply:
done 🙂